src/FormGroup.js

```diff
@@ -39,14 +39,15 @@ const FormGroupWrapper = styled.View`
 `
 
 FormGroupWrapper.defaultProps = {
-  theme: defaultTheme
+  theme: defaultTheme,
+  componentName: 'FormGroupWrapper'
 }
 
 const FormGroup = props => {
   const { border, error, inlineLabel, theme, multiline, numberOfLines, keyboardType, returnKeyType } = props
   const children = React.Children.map(props.children, child => {
     let subsetOfProps = {}
-    if (child.type.name === 'Input') {
```

Comment:
@esbenp In the version before I used this construct to fetch the type of the component. But for some reason this does not work for release mode. The type and some other attributes are not accessible.

But I need the component type for the pass thru of the parent properties that are allowed for the child.